tests/keystore/test_keystore.py

```diff
@@ -72,19 +74,24 @@ def test_key_sqlite_keystore(self):
             key = self.ks.get_key(fingerprint_priv)
 
     def test_keyfrag_sqlite(self):
+        kfrag_component_length = 32
         rand_sig = API.secure_random(65)
-        rand_id = b'\x00' + API.secure_random(32)
-        rand_key = b'\x00' + API.secure_random(32)
+        rand_id = b'\x00' + API.secure_random(kfrag_component_length)
+        rand_key = b'\x00' + API.secure_random(kfrag_component_length)
```

Comment:
This won't be a valid key, fyi. I guess it has a chance of being valid as long as it's within the parameters of the curve, though..

Reply:
Eh, not sure. I think from Ursula's perspective it will be valid.
It has a rekey (big number) and id (which Ursula just attaches). And with any big number, Ursula can even re-encrypt (she'll just get some garbage out)

Reply:
@tuxxy: If you think that there's lacking truth in this test, I'm definitely interested in reforming it.  My efforts were to keep it as close to your pattern as possible while also forcing it to actually test that the de/serialization was happening (not just the storage).

At the moment, this test does shine crucial light on how kFrags are represented as bytes.